driver/pace/driver/diagnostics.py

```diff
@@ -102,9 +110,26 @@ def store(self, time: Union[datetime, timedelta], state: DriverState):
                 except AttributeError:
                     quantity = getattr(state.physics_state, name)
                 zarr_state[name] = quantity
+            derived_state = self._get_derived_state(state)
+            zarr_state.update(derived_state)
             assert time is not None
             self.monitor.store(zarr_state)
 
+    def _get_derived_state(self, state: DriverState):
+        output = {}
+        if len(self.derived_names) > 0:
+            for name in self.derived_names:
+                if "column_integrated_" in name:
+                    tracer = name.split("column_integrated_")[-1]
+                    output[name] = _compute_column_integral(
+                        name,
+                        getattr(state.dycore_state, tracer),
+                        state.dycore_state.delp,
+                    )
+                else:
+                    print(f"{name} is not a supported diagnostic variable.")
```

Comment:
should this be just a print or an error?

Reply:
Changed to warning, my thought is that we would still run the model and just not save any unsupported diagnostic variables.